xapi-rrdd.opam

```diff
@@ -23,4 +23,6 @@ depends: [
   "xapi-stdext-unix"
 ]
 ocaml-version: [>= "4.02.0"]
-
+conflicts: [
+    "fd-send-recv" {<= "2.0.0"}
```

Comment:
Should this be `< 2.0.0` instead, since 2.0.0 is the good version?

Reply:
Yes. Well spotted